src/main/java/jetbrains/buildServer/iaa/utils/UserModelProxy.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2000-2018 JetBrains s.r.o.
+ *
```

Comment:
What is the purpose of this proxy? Why not using UserModelEx directly?

Reply:
From my point of view, with this prozy it, will be much easier to mock this functionality in future, when I will be adding tests.

Reply:
You may either use jmock/Mockery or BaseServerTestCase as a base for the tests. You'll probably need stuff like builds/changes/etc anyway.